# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1233785375 18000
#      Wed Feb 04 17:09:35 2009 -0500
# Node ID 9e0f0a7bf858ecb49266dee36a7eb6a9cdbcc193
# Parent  2a49c32d43228358677d817d4052023fea2b74e7
warning_ids.m: make it a function that just calls help

diff --git a/scripts/miscellaneous/warning_ids.m b/scripts/miscellaneous/warning_ids.m
--- a/scripts/miscellaneous/warning_ids.m
+++ b/scripts/miscellaneous/warning_ids.m
@@ -230,8 +230,12 @@
 ## @code{Octave:undefined-return-values} warning is enabled.
 ## 
 ## @item Octave:variable-switch-label
 ## If the @code{Octave:variable-switch-label} warning is enabled, Octave
 ## will print a warning if a switch label is not a constant or constant
 ## expression.  By default, the @code{Octave:variable-switch-label}
 ## warning is disabled.
 ## @end table
+
+function warning_ids ()
+  help ("warning_ids");
+endfunction
